website/client/src/components/chat/chatCard.vue

```diff
@@ -164,7 +166,7 @@
     .text {
       font-size: 14px;
       color: #4e4a57;
-      text-align: left !important;
+      text-align: left;
```

Comment:
can we keep the !Important here?

Reply:
Or it's not necessary at all given that one of `text-left` or `text-right` is going to be applied anyway?